mesonbuild/scripts/gtkdochelper.py

```diff
@@ -65,6 +65,9 @@ def gtkdoc_run_check(cmd, cwd, library_paths=None):
             library_paths.extend(env['LD_LIBRARY_PATH'].split(os.pathsep))
         env['LD_LIBRARY_PATH'] = os.pathsep.join(library_paths)
 
+    if is_windows():
+        cmd.insert(0, sys.executable)
```

Comment:
Hm, didn't we already do this above `if is_windows() or is_cygwin():`?

Reply:
Huh.  I thought I removed that, since we don't want it in the cygwin case.  Well spotted.

FIxed.